Dockerfile

```diff
@@ -8,11 +8,20 @@ ENV SIGNAL_BUILD_STOP=99 \
     S6_KILL_FINISH_MAXTIME=5000 \
     S6_KILL_GRACETIME=3000
 
-# Slim the container from its pre-installed heft
-RUN apt-get autoclean -y && \
+# Upgrade base packages, then clean packaging leftover
+RUN apt-get update && \
+    apt-get upgrade -yqq && \
+    apt-get install -yqq \
+      curl \
+    && \
+    # Add goss for local testing
+    curl -L https://github.com/aelsabbahy/goss/releases/download/v0.2.3/goss-linux-amd64 -o /usr/local/bin/goss && \
+    chmod +x /usr/local/bin/goss && \
+    apt-get remove --purge -yq curl && \
+    apt-get autoclean -y && \
     apt-get autoremove -y && \
     rm -rf /var/lib/{cache,log}/ && \
-    rm -rf /var/lib/apt/lists/ && \
+    rm -rf /var/lib/apt/lists/* && \
```

Comment:
What about the /var/lib/cache & /var/lib/log dirs, are they ok to delete @bryanlatten ?